pkg/operator/k8s/pod.go

```diff
@@ -108,11 +109,17 @@ func GetPodStatus(pod *corev1.Pod) string {
 		for _, containerStatus := range pod.Status.ContainerStatuses {
 			if containerStatus.LastTerminationState.Terminated != nil {
 				exitCode := containerStatus.LastTerminationState.Terminated.ExitCode
+				if exitCode == 137 {
```

Comment:
Do you think we want these two blocks in the “running” case too, since it can return “killed”?

Reply:
I'm not sure if it would be an OOM case, haven't been able to reproduce to know for sure.